drivers/serial/uart_ra.c

```diff
@@ -366,28 +673,68 @@ static const struct uart_driver_api uart_ra_driver_api = {
 		.clock_id =                                                                        \
 			(clock_control_subsys_t)DT_CLOCKS_CELL_BY_IDX(DT_INST_PARENT(n), 0, id),   \
 		.pcfg = PINCTRL_DT_DEV_CONFIG_GET(DT_INST_PARENT(n)),                              \
+		IF_ENABLED(CONFIG_UART_INTERRUPT_DRIVEN,                                           \
+			   (.irq_config_func = irq_config_func_##n,))}
```

Comment:
I suggest writing this as either

```c
        IF_ENABLED(CONFIG_UART_INTERRUPT_DRIVEN, (
        .irq_config_func = irq_config_func_##n,
        ))
}
```

or

```c
        IF_ENABLED(CONFIG_UART_INTERRUPT_DRIVEN, (
                .irq_config_func = irq_config_func_##n,
        ))
}
```

then it's clear that the `IF_ENABLED` behaves as a `#if` but within a block, what do you think?

Reply:
I modified it according to your 2nd example.